plugins/output-mongodb/src/main/scala/com/stratio/sparkta/plugin/output/mongodb/MongoDbOutput.scala

```diff
@@ -51,25 +49,26 @@ class MongoDbOutput(keyName: String,
 
   override val retrySleep = properties.getString("retrySleep", DefaultRetrySleep).toInt
 
-  override val idAsField = Try(properties.getString("idAsField").toBoolean).getOrElse(false)
-
   override val textIndexFields = properties.getString("textIndexFields", None).map(_.split(FieldsSeparator))
 
   override val language = properties.getString("language", None)
 
-  override def setup: Unit =
+  override def setup : Unit = {
     if (bcSchema.isDefined) {
```

Comment:
what about if schema is not defined? Maybe it should fail if schema is not defined